src/utils/graph/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "@quantumblack/kedro-viz-flowchart",
```

This is just the layout, right?

```suggestion
  "name": "@quantumblack/kedro-viz-layout",
```